test/e2e/storage/testsuites/volumelimits.go

```diff
@@ -109,7 +109,7 @@ func (t *volumeLimitsTestSuite) defineTests(driver TestDriver, pattern testpatte
 	// And one extra pod with a CSI volume should get Pending with a condition
 	// that says it's unschedulable because of volume limit.
 	// BEWARE: the test may create lot of volumes and it's really slow.
-	ginkgo.It("should support volume limits [Slow][Serial]", func() {
```

Comment:
This will be picked up by the job `pull-kubernetes-e2e-gce-csi-serial`.

Reply:
Should we still keep serial? Is it not slow anymore?

Reply:
same question, isn't it still `[Slow]`?